bee-node/src/cli.rs

```diff
@@ -90,7 +94,12 @@ impl ClArgs {
 
     /// Returns whether the node should run as an (autopeering) entry node.
     pub fn run_as_entry_node(&self) -> bool {
-        self.cli.entry_node
+        self.cli.run_as_entry_node
+    }
+
+    /// Returns whether the node should run with enabled autopeering service.
+    pub fn enable_autopeering_service(&self) -> bool {
```

Comment:
Can we just call it `enable_autopeering` for simplicity please ?

Reply:
sure :+1: